vtex-apps/search-graphql
# Add search urls stats

#### What problem is this solving?
Add search urls count stats


We need a way to list the most accessed search urls so we could index and make canonicals to them.

#### How should this be manually tested?
1 - Open [https://jey--storecomponents.myvtex.com/apparel---accessories/shoes/Samsung/Motorola/AOC/LG?map=c,c,b,b,b,b](https://jey--storecomponents.myvtex.com/apparel---accessories/shoes/Samsung/Motorola/AOC/LG?map=c,c,b,b,b,b) or any search at this workspace

2 - You could check the resolver with the stats here:
https://jey--storecomponents.myvtex.com/_v/private/vtex.search-graphql@0.12.1/graphiql/v1


#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
